scripts/create_backup_osgp_dbs.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+HOST=localhost
+PORT=5432
+USER=osp_admin
+
+HOSTNAME=`hostname`
+TIMESTAMP=`date +%Y%m%d_%H%M%S`
+
+BACKUP_DIR=$HOME/backups/osgp/${HOSTNAME}_$TIMESTAMP
```

Comment:
No, the hostname is replaced (backticks) by the actual hostname. At a new dev env, there is no environment HOSTNAME variable.